dds/DCPS/StaticDiscovery.cpp

```diff
@@ -392,16 +392,11 @@ StaticEndpointManager::add_subscription_i(const RepoId& readerid,
     const RepoId& writerid = *pos;
     const EndpointRegistry::Writer& writer = registry_.writer_map.find(writerid)->second;
 
-#ifdef __SUNPRO_CC
-    WriterAssociation wa;
-    wa.writerTransInfo = writer.trans_info;
-    wa.writerId = writerid;
-    wa.pubQos = writer.publisher_qos;
-    wa.writerQos = writer.qos;
```

Comment:
Just curious, are we not supporting this platform anymore?

Reply:
Yes, it's for Solaris, which we don't support anymore.